src/Cardano/Shell/Presets.hs

```diff
@@ -38,7 +38,7 @@ mainnetConfiguration =
           , pcoStaticKeyDlgCertFile     = pure Nothing
           , pcoRequiresNetworkMagic     = pure NoRequireNetworkMagic
           , pcoDBSerializeVersion       = pure 0
-          , pcoPBftSigThd               = pure Nothing
+          , pcoPBftSigThd               = pure 0.22
```

Comment:
Would be nice to have a comment about what that value is and why its there, even just a pointer to elsewhere would be a huge improvement.

Reply:
Also, why are all those `pure`s there on every field?

Reply:
Oh, those `pure` are required by the use of `Last`? Ok, suppose so.

Reply:
Yes, the `pure`'s are due to `Last`, indeed.